modules/ROOT/pages/acm-lightning-components.adoc

```diff
@@ -373,6 +373,19 @@ Extra Field Set:: Specifies extra fields.
 Registrations Require Approval:: Select whether registration requires approval.
 Approver ID:: Specifies the approver ID.
 
+== Spec Downloader
+
+The Spec Downloader component enables users to download an API specification.
+
+The API specification downloaded from API Community Manager is the same as the specification downloaded from Anypoint Exchange.
```

Comment:
How about...:
```suggestion
The Spec Downloader component enables users to download an API specification.

The Spec Downloader component <of this> enables users to download an API specification from API Community Manager that is the same as the Anypoint Exchange specification. In <this UI page>, users can click either of the following:
* *Download Spec as RAML* +
* *Download Spec as OAS* +
```
...but I can't figure out what the rest means. Is the general practice to download to the same type (RAML or OAS) as originally uploaded? And if the two differ, then "the downloaded version is automatically generated"? What does that mean, that it is automatically generated; what is the alternative? And why would an automatically generated version be different from its source?

If this is clear to your intended audience, then ignore my confusion. But look at this text again and be sure that the wording you use "connects the dots" and does not rely on audience assumptions.

Reply:
This is explained in more detail in the Exchange docs, and will be somewhat clearer to the audience, but it's a good point, and I added some more explanation. Users can upload either type, and the other type will be generated automatically so that both are available. The alternative would be for one of them to be missing. The differences from the source are caused by unavoidable differences in the RAML and OAS formats.